packages/@react-native-windows/codegen/package.json

```diff
@@ -15,7 +15,7 @@
     "watch": "rnw-scripts watch"
   },
   "bin": {
-    "react-native-windows-codegen": "./bin.js"
+    "@react-native-windows/codegen": "./bin.js"
```

Comment:
I don't think you want this change... As this will change the name customers use in their 'scripts' section...

Reply:
We only use this internally at build time right now. See just-scripts in vnext for where invoked